### PR TITLE
loading screen update

### LINKED ISSUE #1984: Improve loader
Our current loading placeholder is not bad by any stretch of the imagination but, we can tweak it a little bit.

### PR DESCRIPTION
**Pull request by schalkneethling**
- switch to dino for loading screen
- fix: update page loading state

Fixes #1984

### CODE REVIEW

**Review by mattiapontonio — COMMENTED** [review]
Should be AWSome 😌 if possibile ti see the link before the merge.
Anyway 👍🙂

File: client/src/document/index.scss [hunk 1]
```diff
@@ -3,15 +3,29 @@
...
 
+@import "~@mdn/minimalist/sass/mixins/animation";
 @import "~@mdn/minimalist/sass/mixins/utils";
 
 @import "./prism-overrides";
 
-.loading-document-placeholder article {
-  // Any huge margin bottom to keep the footer away when loading
-  margin-bottom: 1200px;
+/* we already disable all animation and transitions globally via a reset so */
```
Comment (peterbe) [inline client/src/document/index.scss:11]: I don't understand this comment. Does it make sense to you ...in 6months time?

File: client/src/assets/dino.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" role="img"><defs><style>.cls-1{fill:#eee;}</style></defs><g id="Layer_1" data-name="Layer 1"><path class="cls-1" d="M10.56,49.41H25.69L16.9,58.72H29.06l-3.89,10.2C40.06,84.16,55.93,87.23,55.93,87.23c0-4.2,1-20.69,2.07-23.45a13.38,13.38,0,0,1,1.91-3.86A10.58,10.58,0,0,1,68.43,56,27,27,0,0,1,78,57.79a6.57,6.57,0,0,0,5.72-.3c1.27-.76,2.53-1.55,3.76-2.36a2.06,2.06,0,0,1,1.39-.45,3.1,3.1,0,0,0,3.19-1.85c.4-.83.88-1.6,1.32-2.43A13.77,13.77,0,0,0,95,43.17h0c-.1-1.81-.31-3.63-.43-5.43a4.4,4.4,0,0,0-1-2.46A6.12,6.12,0,0,0,91,33.42c-2-.79-4.07-1.67-6.11-2.45a54,54,0,0,1-8.39-3.69,6.61,6.61,0,0,1-3.26-5,7.39,7.39,0,0,0-3.69-5.59,10.19,10.19,0,0,0-7.42-.88,11.3,11.3,0,0,1-5.6-.08l-2.79-.85L53,14.69,50.78,14c-4-1.18-5.6-1.79-12-.7A36.94,36.94,0,0,0,21,23.12l-16,17H19.36Z"/></g></svg>
```
Comment (peterbe) [inline client/src/assets/dino.svg:1]: Why is this styled here? Why not in `document/index.scss`?

File: client/src/document/index.scss [hunk 3]
```diff
@@ -3,15 +3,29 @@
...
+    @media #{$mq-small-desktop-and-up} {
+      margin: $base-spacing auto;
+      max-width: $max-width-default;
+      min-height: 75vh;
+      width: 50%;
+    }
+  }
 }
 
+/* stylelint-enable */
```
Comment (peterbe) [inline client/src/document/index.scss:28]: Shouldn't this comment sit immediately after the `}` bracket for the `&.loading-document-placeholder {...}`  block?

**Review by peterbe — APPROVED** [review]
With nits/questions.

File: client/src/document/index.scss (hunk 1 above)
Comment (schalkneethling) [reply client/src/document/index.scss:11]: Yup 😄 it's the reset used in minimalist. I can totally add that though, for additional context.

File: client/src/assets/dino.svg (hunk 2 above)
Comment (schalkneethling) [reply client/src/assets/dino.svg:1]: Ah yes, it was not going to be inlined SVG originally. Nice catch. Will update.

File: client/src/document/index.scss (hunk 3 above)
Comment (schalkneethling) [reply client/src/document/index.scss:28]: Turns out, as I am adding the animation via a mixin now, the code has linter has no idea I am even using `animation` and so those comments are no longer needed :+1:

**Pull request merged**